dashboard/test/controllers/api/v1/pd/course_facilitators_controller_test.rb

```diff
@@ -1,27 +1,17 @@
 require 'test_helper'
 
 class Api::V1::Pd::CourseFacilitatorsControllerTest < ::ActionController::TestCase
-  test 'admins can see course facilitators' do
-    sign_in create(:admin)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :success
-  end
-
-  test 'workshop organizers can see course facilitators' do
-    sign_in create(:workshop_organizer)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :success
-  end
-
-  test 'facilitators cannot see course facilitators' do
-    sign_in create(:facilitator)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :forbidden
-  end
-
-  test 'teachers cannot see course facilitators' do
-    sign_in create(:teacher)
-    get :index, params: {course: Pd::Workshop::COURSES.first}
-    assert_response :forbidden
+  {
+    admin: :success,
+    workshop_organizer: :success,
+    facilitator: :forbidden,
+    teacher: :forbidden
+  }.each do |user_type, response|
+    test_user_gets_response_for(
+      :index,
+      user: user_type,
+      response: response,
+      params: {course: Pd::Workshop::COURSES.first}
+    )
```

Comment:
Ha, perhaps it is. What do you think of looping through a set of users, such as [this](https://github.com/code-dot-org/code-dot-org/pull/13734/files#diff-79a40890d00d79ffb69a17deb85d7556R9)? Would you prefer I split this up into two such loops, one each for success and forbidden responses? Or split fully into 4 blocks? This last then seems not DRY enough in my opinion, but I like the middle option. Thoughts?